test/examples_test.go

```diff
@@ -271,7 +271,7 @@ func imageNamesMapping() map[string]string {
 			"registry":                              getTestImage(registryImage),
 			"node":                                  "node:alpine3.11",
 			"gcr.io/cloud-builders/git":             "alpine/git:latest",
-			"docker:dind":                           "ibmcom/docker-s390x:20.10",
+			"docker@sha256:74e78208fc18da48ddf8b569abe21563730845c312130bd0f0b059746a7e10f5":                           "ibmcom/docker-s390x:20.10",
```

Comment:
You need to fix formatting with `gofmt`. https://prow.tekton.dev/view/gs/tekton-prow/pr-logs/pull/tektoncd_pipeline/7600/pull-tekton-pipeline-build-tests/1749843714966032384